migrations/committed/000052.sql

```diff
@@ -0,0 +1,32 @@
+--! Previous: sha1:fd9c0b6eecc2142dce19df696187ebce98fca01e
+--! Hash: sha1:8e3184e177d3b8f70ed3800152786830ae61f705
+
+CREATE FUNCTION anchor_iri() RETURNS trigger AS $$
```

Comment:
just another thing I realized we need to update before merging this: it's best to have idempotent migrations (https://github.com/graphile/migrate/blob/main/docs/idempotent-examples.md) so we need to add the following I think:
```suggestion
DROP FUNCTION IF EXISTS anchor_iri CASCADE;
CREATE FUNCTION anchor_iri() RETURNS trigger AS $$
```

Reply:
Done in https://github.com/regen-network/regen-server/pull/454/commits/a206fe1661527c9e5248ca6676308e1305c98cf1